prescription.sh

```diff
@@ -233,6 +241,10 @@ function generateYML () {
 	    s~<<JIRA_ISSUES_QUERY>>~$jira_issues_query~g; \
 	    s~<<JIRA_USERNAME>>~$jira_username~g; \
 	    s~<<JIRA_AUTH_TOKEN>>~$jira_auth_token~g; \
+            s~<<RALLY_PROJECT_NAME>>~$rally_project_name~g; \
```

Comment:
missing indentation here as well

Reply:
indentation looks good on the branch -  here it shows that way
https://github.com/synopsys-sig/io-artifacts/blob/POLCT-157/prescription.sh

Reply:
Not sure might why it shows as missed indentation here, but if the indentation is correct in main code then its fine